### PR TITLE
[Doc Improvement][API ME Teamsfx update][3948834]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c92e63d5a9cb4ef1fddd946f2b788b609749d1da/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5e3d8f00-20a9-4d24-a224-38e3fea2e78d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c92e63d5a9cb4ef1fddd946f2b788b609749d1da/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d5f93872-a7ca-4831-862e-2564e12d5f39) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/c92e63d5a9cb4ef1fddd946f2b788b609749d1da/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 79 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c3bf3480-eb06-4eb3-a820-dd9f293e3117) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df72590daa2fd101d4a9b518e6045fee48ca8f3b/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41865bb6-954c-4661-bd50-45ca3f4c2237) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/df72590daa2fd101d4a9b518e6045fee48ca8f3b/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ff419db8-0687-4797-9f24-4873d18a6872) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/df72590daa2fd101d4a9b518e6045fee48ca8f3b/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 79 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/97914b00-83aa-4af0-8fe1-20f5340c35b2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-api-msg-ext-ttk.yml [hunk 1]
```diff
@@ -174,27 +174,27 @@ items:
...
         ```
 
         Your default web browser opens to let you sign in to the account. Sign in to your Azure account using your credentials. Close the browser when you're prompted.
 
-    2. Sign in to Azure with the TeamsFx CLI:
+    2. Sign in to Azure with the Teamsapp CLI:
 
         ```cmd
 
-        teamsfx account login azure
```
Comment (Wajeed-msft) [inline msteams-platform/sbs-api-msg-ext-ttk.yml:191]: Please check if we are using CLI in any steps by step guides.

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d4b22fb626472345accbbb4eb947518ce0f65e65/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0e845fb3-a52f-415a-b4c8-4f96a22dd9c5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d4b22fb626472345accbbb4eb947518ce0f65e65/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80d1b2c8-13c0-4773-a186-5987de0870f6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/d4b22fb626472345accbbb4eb947518ce0f65e65/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 79 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/548a0571-dd4c-49e4-aa17-547763f49392) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/sbs-api-msg-ext-ttk.yml (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/sbs-api-msg-ext-ttk.yml:191]: Sure, will check.

**Review by Wajeed-msft — DISMISSED** [review]

**Review by summzhan — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b59eeb1b7566562974b6ce4180dce7e5811af8c2/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b992d10c-fb63-4fc7-bf63-eb308979de17) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b59eeb1b7566562974b6ce4180dce7e5811af8c2/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1fa0dd7e-c049-4f5c-9479-8f9a122ec59b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/b59eeb1b7566562974b6ce4180dce7e5811af8c2/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 79 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b5973ea7-05f8-4012-ba69-7c24db0591fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a12546676dfd804972c5dbaf421967e0ac7324d8/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4251a3e0-2d23-4480-89f2-6cd79feabcfb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a12546676dfd804972c5dbaf421967e0ac7324d8/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6f12534e-8401-4095-ba63-07dcfd842e61) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/a12546676dfd804972c5dbaf421967e0ac7324d8/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 79 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f16b06ba-bb7e-4356-bc4a-e9c47dd438d0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/messaging-extensions/build-bot-based-plugin.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7ef51db09a6bf367df03cc0cce1983ceab15d8ba/msteams-platform%2Fmessaging-extensions%2Fbuild-bot-based-plugin.md) | 94 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b2476969-b55a-4755-83ba-3353efc7659a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/messaging-extensions/create-api-message-extension.md](https://github.com/MicrosoftDocs/msteams-docs/blob/7ef51db09a6bf367df03cc0cce1983ceab15d8ba/msteams-platform%2Fmessaging-extensions%2Fcreate-api-message-extension.md) | 96 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cea9eeda-49ad-4c98-848a-bb5c273fb945) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/sbs-api-msg-ext-ttk.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/7ef51db09a6bf367df03cc0cce1983ceab15d8ba/msteams-platform%2Fsbs-api-msg-ext-ttk.yml) | 81 | 79 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1aa236d6-7879-49b7-a1bb-4793392e07e4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**